src/Sign.Core/Signer.cs

```diff
@@ -104,7 +104,7 @@ public Signer(IServiceProvider serviceProvider, ILogger<ISigner> logger)
                     else
                     {
                         // if the output is specified, treat it as a directory, if not, overwrite the current file
-                        if (!string.IsNullOrWhiteSpace(outputFile))
+                        if (string.IsNullOrWhiteSpace(outputFile))
```

Comment:
During the conversion to a standalone client, it looks the semantics of the [original expression](https://github.com/dotnet/sign/blob/34f0acefee3673ba192ae0f72e39d029d4d2ef78/src/SignClient/SignCommand.cs#L287) weren't preserved.  Good catch.